src/Sarif/Sarif.csproj

```diff
@@ -39,6 +39,7 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="GlobalSuppressions.cs" />
```

Comment:
(Complete aside: UTF-16 files are terrible. Is there a bug filed on the VS integration to use UTF-8?)